github/src/main/kotlin/gropius/sync/github/IMSConfig.kt

```diff
@@ -1,15 +1,17 @@
 package gropius.sync.github
 
 import gropius.model.architecture.IMS
+import java.net.URI
 
 /**
  * Config read out from a single IMS node
  * @param ims the gropius ims to use as input
  * @param botUser the bot user string extracted from the template
  * @param readUser the read user string extracted from the template
+ * @param url the read url extracted from the template
  */
 data class IMSConfig(
-    val ims: IMS, val botUser: String, val readUser: String
+    val ims: IMS, val botUser: String, val readUser: String, val graphQLUrl: URI
```

Comment:
documentation name does not fit to param name

Reply:
705ec38fdc40f8ca96b5c7b883c1f841eb66a176